tfe/data_source_organization_membership.go

```diff
@@ -40,28 +40,44 @@ func dataSourceTFEOrganizationMembershipRead(d *schema.ResourceData, meta interf
 	// Create an options struct.
 	options := &tfe.OrganizationMembershipListOptions{
 		Include: []tfe.OrgMembershipIncludeOpt{tfe.OrgMembershipUser},
+		Emails:  []string{email},
```

Comment:
Yes and it would behave as expected, i.e letting the user know that the org user with an empty email does not exist -- either by returning zero org members or by brute searching against all org members and checking if `orgMember.Email == ""` (obviously will never return a match)